src/Service/NotificationReceiverService.php

```diff
@@ -122,7 +123,8 @@ public function process(Request $requestObject)
 
             // Process each notification item
             foreach ($request['notificationItems'] as $notificationItem) {
-                if (!$this->processNotificationItem($notificationItem['NotificationRequestItem'])) {
+                $notificationItem['NotificationRequestItem']['live'] = $request['live'];
```

Comment:
why is this set here?

Reply:
This was one of the bugs found when testing, `live` is needed when [creating the `NotificationEntity`](https://github.com/Adyen/adyen-shopware6/blob/develop/src/Service/NotificationService.php#L102) and it is not set inside the `NotificationRequestItem` object of the request ([see request format in docs](https://docs.adyen.com/development-resources/webhooks/understand-notifications?tab=codeBlockOU4K1_1#notification-structure))